Polygon_mesh_processing/examples/Polygon_mesh_processing/hausdorff_distance_remeshing_example.cpp

```diff
@@ -4,7 +4,7 @@
 #include <CGAL/Polygon_mesh_processing/distance.h>
 #include <CGAL/Polygon_mesh_processing/remesh.h>
 
-#if CGAL_LINKED_WITH_TBB
+#if defined(CGAL_LINKED_WITH_TBB) && !defined(CGAL_NO_ATOMIC)
```

Comment:
If an implementation uses TBB, then it can use TBB atomic types, instead of those provided by the STL or Boost.